src/extensions/bigbang/manifests.go

```diff
@@ -91,7 +91,7 @@ func manifestGitRepo(cfg *extensions.BigBang) fluxSrcCtrl.GitRepository {
 	return fluxSrcCtrl.GitRepository{
 		TypeMeta: metav1.TypeMeta{
 			Kind:       fluxSrcCtrl.GitRepositoryKind,
-			APIVersion: "source.toolkit.fluxcd.io/v1beta2",
```

Comment:
I do think it would be good to do some backwards compatibility handling here - it shouldn't be too difficult to check the semver since you have the full extension configuration.

Reply:
Let me know if I addressed this well enough.